### PR TITLE
Add multi faceting for u2f

### PR DESCRIPTION
**Pull request by mitsuhiko**

### CODE REVIEW

File: src/sentry/web/frontend/twofactor.py [hunk 1]
```diff
@@ -142,3 +143,18 @@ def handle(self, request):
             'other_interfaces': self.get_other_interfaces(interface, interfaces),
             'activation': activation,
         }, request, status=200)
+
+
+def u2f_appid(request):
```
Comment (mattrobenolt) [inline src/sentry/web/frontend/twofactor.py]: Wouldn't we want this to be behind auth?

**Comment by mattrobenolt** [comment]
ok

File: src/sentry/web/frontend/twofactor.py (hunk 1 above)
Comment (mitsuhiko) [inline src/sentry/web/frontend/twofactor.py]: Needs to be public as per https://fidoalliance.org/specs/fido-u2f-v1.0-ps-20141009/fido-appid-and-facets-ps-20141009.html

Comment (mitsuhiko) [inline src/sentry/web/frontend/twofactor.py]: In particular even if we would protect it:

> The URL must be dereferenced with an anonymous fetch. That is, the HTTP GET must include no cookies, authentication, Origin or Referer headers, and present no TLS certificates or other forms of credentials.

**Pull request merged**